WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/SimplePaymentsMigrationView.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+
+/// Shows information about the migration from simple payments to order creation in Menu tab > Payments > Collect Payment.
+struct SimplePaymentsMigrationView: View {
+    let addCustomAmount: () -> Void
```

Comment:
Question: Since this doesn't have to be internal, would it be worth to make it private and add an initializer as:
```suggestion
    private let addCustomAmount: () -> Void
    
      init(addCustomAmount: @escaping () -> Void) {
        self.addCustomAmount = addCustomAmount
    }
    
```

Or not really necessary?

Reply:
Yup good suggestion it's better to mark it private with an initializer, updated in https://github.com/woocommerce/woocommerce-ios/pull/12400/commits/e8bf71488f39203f7cf10c403d9d3919466d4a76.